test/es6_spec.coffee

```diff
@@ -329,6 +329,28 @@ describe "JavaScript Test Suite", ->
       expect(gen.next().done).toEqual true
       expect(dude.enemy).toEqual "slain!"
 
+    it "Named fn expr", ->
+      dude =
+        slay: -> @enemy = "slain!"
+        hesitate: -> aether._shouldYield = true
+      code = """
+        this.f = function named() {
+          this.hesitate();
+          this.hesitate();
+          this.hesitate();
+        };
+        this.f();
+        this.slay();
+      """
+      aether.transpile code
+      f = aether.createFunction()
+      gen = f.apply dude
```

Comment:
Oh, it is because I've fixed `makeYieldAutomatically` only and this test uses `makeYieldConditionally`. Gawd I'm dumb.